FrontMen/fm-hours
# Feat/intracto auth

# Changes

## Related issues

Resolves #121 

## Added

- Token logic
  - PPid expires every 7 days
  - Auth token expires every 24 hours

## Removed

N/A

## Changed

- Update employees to have the `bridgeUid`

## How to test

- Log in
- Refresh the app
- Navigate through the pages
- Log out
- Make sure that all the steps above work as expected 
- The user should only be automatically logged out if the `PPid` is expired

## Screenshots

N/A
